orchestration/dagster_orchestration/hca_utils/utils.py

```diff
@@ -313,3 +323,5 @@ def _process_rows(self, get_table_names, get_rids, soft_delete: bool, issue: str
                     finally:
                         # delete file
                         os.remove(local_filename)
+                problem_count += len(rids_to_process)
```

Comment:
🤔 i wonder if we could rewrite this more functionally, e.g. `[get_rids(table_name) for table_name in table_names]` and then running the soft-delete and problem-counting steps as additional operations on the `get_rids` result.